habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -428,31 +428,31 @@ def action_space(self):
     def apply_ee_constraints(self):
         self.ee_target = np.clip(
             self.ee_target,
-            self._sim.robot.params.ee_constraint[:, 0],
-            self._sim.robot.params.ee_constraint[:, 1],
+            self._sim.agent.params.ee_constraint[:, 0],
```

Comment:
Thanks for this! I was thinking about should it be something like `self._sim.agent.robot.params.ee_constraint[:, 0]`? This is because that robot class lies below agent class. Please correct me if I miss something, thank you!

Reply:
I think we would want to be able to apply ee_constraints for both robots and other types of agents, so this is more of a renaming code change. `self._sim.agent` contains a MobileManipulator agent, which could either be a robot or a human.

Reply:
I got, thank you!